src/facade/reply_builder.cc

```diff
@@ -212,7 +212,11 @@ MCReplyBuilder::MCReplyBuilder(::io::Sink* sink) : SinkReplyBuilder(sink), all_(
 }
 
 void MCReplyBuilder::SendValue(std::string_view key, std::string_view value, uint64_t mc_ver,
-                               uint32_t mc_flag) {
+                               uint32_t mc_flag, bool send_mc_ver) {
+  // Return a placeholder cas token of 0
+  if (send_mc_ver) {
```

Comment:
this is not needed. Just pass 0 at caller

Reply:
There is a check later on which does not write version if it is 0, should I remove it for now? 

The boolean is mainly for that check https://github.com/dragonflydb/dragonfly/blob/bdfa30d77b303cdf01627aea17257032f39c679d/src/facade/reply_builder.cc#L234

Reply:
The boolean parameter might still be needed, the same code path is used for `GET` and `GETS`. It is used to differentiate if the version has to be written or not.

Reply:
yes, keep boolean, just rename it to "send_cas" but you do not need to override `mc_ver` - it's confusing. just pass 0 at the caller